public/models/interfaces.ts

```diff
@@ -184,7 +197,20 @@ export type AnomalySummary = {
   lastAnomalyOccurrence: string;
 };
 
+export type HistoricalDetectorAnomalySummary = {
+  anomalyOccurrence: number;
+  minAnomalyGrade: number;
+  maxAnomalyGrade: number;
+  minConfidence: number;
+  maxConfidence: number;
+};
+
 export type DateRange = {
   startDate: number;
   endDate: number;
 };
+
+export type DetectionDateRange = {
```

Comment:
I think re-using `DateRange` should be sufficient.

Reply:
The reason for adding this `DetectionDateRange` is because the fields understood by the backend take `startTime` and `endTime` fields instead of `startDate` and `endDate` fields used by the `DateRange`. I didn't want to add more converting when parsing existing detectors or creating/updating new detectors. Let me know what you think.